doc/manual/src/glossary.md

```diff
@@ -34,6 +34,10 @@
     the same output. This cannot be guaranteed in general (e.g., a
     builder can rely on external inputs such as the network or the
     system time) but the Nix model assumes it.
+    
+  - Nix database\
+    a sqlite database in contained in the Nix store which contains metadata 
```

Comment:
"in contained in" is not grammatical. Also, the DB is not *in* the Nix store but elsewhere (/nix/var/nix/db).

"a" should be capitalized. "sqlite" should be "SQLite".

Reply:
```suggestion
    An SQlite database to track references between store objects. Default location: `/nix/var/nix/db`.
```